lib/errors_savior/protocol/render_error.rb

```diff
@@ -0,0 +1,34 @@
+module ErrorsSavior
+  module Protocol
+    class RenderError
+      def initialize(savior, error)
+        @savior = savior
+        @error = error
+      end
+
+      def as_json
+        json = required_error_body
+        json['errors'] = errors.as_json if @savior.respond_to?(:errors)
+
+        Rails.configuration.consider_all_requests_local ? append_metadata(json) : json
+      end
+
+      def required_error_body
+        {
+          error_code: @savior.error_code,
+          message: @savior.message,
+          timestamp: Time.zone.now
+        }
+      end
+
+      def append_metadata(json)
+        json['metadata'] = {}
+        json['metadata']['error_class'] = @savior.error_class.name
+        # TODO: json['metadata']['api_origin'] = ErrorsSavior.config.api_origin.enabled?
```

Comment:
Mmmm comment code is not the best, maybe you can describe the functionality that you want here ;)

Reply:
:eyes: I fix this line [in the next PR](https://github.com/Wolox/errors_savior/pull/5)